content/hardware/07.opta/opta-family/opta/tutorials/18.conveyor-sorting-app-note/content.md

```diff
@@ -0,0 +1,621 @@
+
+![Conveyor and Sorting System](assets/overview-diagram.png)
+
+### Sensors Deployment
+
+- The **RGB Color Sensor** will be responsible for detecting the color of the item. It will be placed at the system's entry, so the color of the item will be detected first by this sensor, allowing the item to be sorted accordingly to its color.
+- The **Photoeletric Sensor** this sensor will be responsible for detecting the presence of the item at the separation position, activating the actuator responsible for separating the item. Three of these sensors will be used, one for each color of the items being separated, placed at the separation position.
+
+### Actuators Deployment
+
+- The **Solid State Relay** is used to activate loads that are not supported by the PLC output. This way, the PLC can control equipment requiring higher load capacity, such as activating the two conveyor motors for item movement and the two item separation actuators.
+
+## Step-by-Step Guide for System Operation
+
+1. **System Start**
+   - **Action:** Press the Start button.
+   - **Description:** The system is activated, initiating the operation of both the feeder conveyor and the sorting conveyor.
+
+2. **Activation of Conveyors**
+   - **Action:** The feeder and sorting conveyors start operating simultaneously.
+   - **Description:** The feeder conveyor moves objects from the initial loading point toward the RGB color sensor area.
+
+3. **Color Detection**
+   - **Action:** Objects move along the feeder conveyor until the RGB color sensor detects a specific color.
+   - **Description:** The RGB color sensor detects the color of the object on the feeder conveyor.
+
+4. **Action After Detection**
+   - **Action:** The feeder conveyor continues to run briefly after color detection.
+   - **Description:** The feeder conveyor remains active for a short period to ensure the object successfully transfers to the sorting conveyor.
+
+5. **Actuator Activation**
+   - **Action:** Activate the specific actuator corresponding to the detected color.
+   - **Description:** The actuator is triggered, creating a physical barrier that pushes the object off the sorting conveyor and into the designated area.
+
+6. **Sorting Confirmation**
+   - **Action:** Confirm the object has been sorted.
+   - **Description:** A photoelectric sensor, positioned in the designated area for each color, confirms that the object has been successfully sorted.
+
+7. **System Stop**
+   - **Action:** Press the stop button.
+   - **Description:** The system is stoped, stopping all operations and deactivating the conveyors and actuator.
+
+8. **Remote Monitoring and Control**
+   - **Action:** Monitor and control the system via the Arduino Cloud.
+   - **Description:**  At any time, you can check the operating status and control the system remotely using the Arduino Cloud interface
+
+## Opta Ladder Code
```

Comment:
Would be nice to have a screenshot of the PLC IDE so reader can mentally locate in the software where the Ladder diagram will be written.